Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -69,6 +95,8 @@ internal string EffectiveProfileQueryEndpoint
         /// </summary>
         public void OnBeginRequest(HttpContext context)
         {
+            this.childRequestTrackingSuppressionModule?.OnBeginRequest_IdRequest(context);
```

Comment:
I think you can safely move this code to after `this.initializationErrorReported` check.